### PR TITLE
Added a basic enemy to test projectile damage

### PR DESCRIPTION
**Pull request by DLMcPherson**

### CODE REVIEW

File: source/scripts/Bullet.js [hunk 1]
```diff
@@ -77,6 +80,16 @@ export default class Bullet extends Pixi.Sprite {
     }
     collideWithBaddies() {
         // TODO: https://github.com/ehgoodenough/gmtk-2017/issues/4
+        if(this.parent != undefined) {
+            this.parent.children.forEach((child) => {
+                if(child instanceof Foe) {
+                    if(getDistance(child.position,this.position) < 20 && child.hearts > 0){
```
Comment (ehgoodenough) [inline source/scripts/Bullet.js:86]: Can we move 20 to a constant, like `HARM_RADIUS` or something?

File: source/scripts/Bullet.js [hunk 2]
```diff
@@ -77,6 +80,16 @@ export default class Bullet extends Pixi.Sprite {
     }
     collideWithBaddies() {
         // TODO: https://github.com/ehgoodenough/gmtk-2017/issues/4
+        if(this.parent != undefined) {
```
Comment (ehgoodenough) [inline source/scripts/Bullet.js:83]: Nice! It's good to check if the parent exists, just in case.

File: source/scripts/Scene.js [hunk 3]
```diff
@@ -10,6 +11,7 @@ export default class Scene extends Pixi.Container {
         super()
 
         this.addChild(this.player = new Player())
+        this.addChild(this.foe = new Foe())
```
Comment (ehgoodenough) [inline source/scripts/Scene.js:14]: Would you mind renaming it to `Baddie`, so it matches our other calls about baddies? Thoughts?

File: source/scripts/Bullet.js (hunk 1 above)
Comment (DLMcPherson) [reply source/scripts/Bullet.js:86]: I'm good with that

File: source/scripts/Scene.js (hunk 3 above)
Comment (DLMcPherson) [reply source/scripts/Scene.js:14]: GHYE GHE GHE HE HE HE. Sure, bro. I was gonna call them "Enemy" but I didn't like the sound of that. I'm glad you also have a preferred name to Enemy XD

**Comment by DLMcPherson** [comment]
### Summary

* Created Enemy class as an inheritor of PIXI.Sprite (modeled after Player class)
* Added Enemy sprite to Scene
* Added collision check function for projectiles with enemies

### Playable Link
https://ehgoodenough.github.io/gmtk-2017/enemy

### Animated Screenshot
enemybranch

**Pull request merged**